ola.spec

```diff
@@ -1,91 +1,109 @@
-BuildRequires: libmicrohttpd-devel >= 0.4.0, libcppunit-devel, protobuf-devel >= 2.1.0
-Requires:      libmicrohttpd >= 0.4.0, libcppunit, protobuf >= 2.1.0
+Name:           ola
+Version:        0.9.7
+Release:        1
+Summary:        Open Lighting Architecture
+Group:          Applications/Multimedia
+License:        GPLv2 and LGPLv2
+URL:            https://github.com/OpenLightingProject/ola
+Source:         https://github.com/OpenLightingProject/ola/releases/download/%{version}/%{name}-%{version}.tar.gz
+BuildRoot:      %{_buildrootdir}/%{name}-%{version}-%{release}-root
+BuildRequires:  libmicrohttpd-devel,cppunit-devel,protobuf-devel,protobuf-compiler,protobuf-python,libftdi-devel,openslp-devel,uuid-devel,libtool,bison,flex,pkgconfig,gcc,gcc-c++,python-devel,avahi-devel
 
 %description
-The Open Lighting Architecture (OLA) consists of two parts, the daemon olad and the library, libola.
-
-
-%package -n libola
-Group:         Other
-Summary:       OLA - Open Lighting Architecture
-
-%description -n libola
-The OLA library
-
-
-%package -n libola-devel
-Group:         Other
-Summary:       OLA - Open Lighting Architecture
-
-%description -n libola-devel
-The OLA library headers
-
+The Open Lighting Architecture is a framework for lighting control information.
+It supports a range of protocols and over a dozen USB devices. It can run as a
+standalone service, which is useful for converting signals between protocols,
+or alternatively using the OLA API, it can be used as the back-end for lighting
+control software. OLA runs on many different platforms including ARM, which
+makes it a perfect fit for low cost Ethernet to DMX gateways.
+
+%package devel
+Requires:      ola = %{version}-%{release}, protobuf-devel
+Group:         Development/Libraries
+Summary:       C/C++ Development files for OLA
+
+%description devel
+The OLA C/C++ library
+
+%package -n python2-%{name}
+Requires:      ola = %{version}-%{release}, protobuf-python, python(abi) = 2.7
+Group:         Development/Libraries
+Summary:       Python Development files for OLA
+BuildArch:     noarch
+%{?python_provide:%python_provide python2-%{name}}
+
+%description -n python2-%{name}
+The OLA python library
+
+%package rdm-tests
+Requires:      ola = %{version}-%{release}, python2-%{name}
+Group:         Development/Libraries
+BuildArch:     noarch
+Summary:       RDM test suite using OLA and python
+
+%description rdm-tests
+The rdm test suite for OLA
 
 %prep
-%setup -q -n %{name}
-
+%setup -q -n %{name}-%{version}
 
 %build
-%define _disable_ld_no_undefined 1
-export LDFLAGS="-Wl,-undefined -Wl,dynamic_lookup"
 autoreconf -i
-%configure
-%make
+%configure --enable-rdm-tests --enable-shared --disable-static
+sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
+sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
+sed -i "s|\$(LN_S) -f \$(bindir)\/|\$(LN_S) -f |g" Makefile
+make %{?_smp_mflags}
```

Comment:
the ```%{?_smp_mflags}``` typically adds the ```-j4``` to make, however epel 6 doesn't handle this very well and the build errors out. should we include this?